products/apache-kafka.md

```diff
@@ -30,6 +30,13 @@ auto:
 
 # eol(x) = MAX(latestReleaseDate, releaseDate(X+1))
 releases:
+-   releaseCycle: "4.0"
+    releaseDate: 2025-03-18
+    eol: false
+    #eoes: 2027-02-19
```

Comment:
```suggestion
    eoes: false
```

Extended support is systematic, will be automatically updated when https://docs.confluent.io/platform/current/installation/versions-interoperability.html will document it.